docs/database/database-spans.md

```diff
@@ -63,9 +63,11 @@ Semantic conventions for individual database systems MAY specify different span
 The <span id="target-placeholder">`{target}`</span> SHOULD describe the entity that the operation is performed against
 and SHOULD adhere to one of the following values, provided they are accessible:
 
-- `db.collection.name` SHOULD be used for data manipulation operations or operations on a database collection.
+- `db.collection.name` SHOULD be used for operations on a specific database collection.
+- `db.stored_procedure.name` SHOULD be used for operations that call a specific stored procedure.
```

Comment:
would we populate it for DDL? Creating/updating/removing stored procedure? Maybe

```suggestion
- `db.stored_procedure.name` SHOULD be used for operations on a specific stored procedure.
```